src/HardSphereDynamics.jl

```diff
@@ -12,6 +12,13 @@ using StaticArrays
 using Parameters
 using Distributions
 
+using Requires
+
+
+function __init__()
+    @require Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a" include("makie_visualization.jl")
```

Comment:
```suggestion
    @require Makie="ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a" include("makie_visualization.jl")
```
Needs to have no spaces, limitation of macros I think